src/panel/theme.ts

```diff
@@ -3,7 +3,8 @@ export const theme = {
     "-1": "#607E90",
     "0": "#AAADB1",
     "+1": "#A5B0B7",
-    "+2": "#b4bfd1"
+    "+2": "#b4bfd1",
+    "+3": "#CECECE"
```

Comment:
So you're in the loop, the goal was to have a linear scale for each color looking something like like this

```
-2: darken(blue, 0.4)
-1: darken(blue, 0.2)
0: blue,
+1: brighten(blue, 0.2)
+2: brighten(blue, 0.4)
```

Since adding in the cache viewer and new color schemes, colors are a bit haywire at the moment but hopefully we can get them to be back to that linear scale eventually but for now this is fine!

Reply:
Good point - I do remember you mentioning that. I can change this to be `tempGray` for the time being so we remember to change it?